doc/modules/ROOT/pages/contribute/coding-guidelines.adoc

```diff
@@ -34,6 +34,7 @@
 * Add comments to code that is difficult to understand
 * Prefer functional programming (e.g. use Lambda expressions)
 * Add JUnit tests.
+* Run ./gradlew :io.openems.edge.io.shelly:clean in the root Directory of openEMS to check the tests of your module.
```

Comment:
The official spelling is **OpenEMS** - at least _we_ should stick to it.

"directory" should be lower-case here.

If fact I believe this note would better fit in https://github.com/OpenEMS/openems/blob/develop/doc/modules/ROOT/pages/edge/implement.adoc than in the general coding-guidelines.

Reply:
Under which Section do you think it is better suited ?

Reply:
Maybe just add it as a separate chapter `1.11`?!